src/article/article.ts

```diff
@@ -70,3 +73,31 @@ const getHeadings = (articleDom: DocumentFragment): Heading[] => {
     return headings;
   }, new Array<Heading>());
 }
+
+const getHeader = (articleDom: DocumentFragment): string => {
+  const headline = articleDom.querySelector('article > [itemprop="headline"]');
+  const authors = articleDom.querySelector('article > [data-itemprop="authors"]');
+  const affiliations = articleDom.querySelector('article > [data-itemprop="affiliations"]');
+  const publisher = articleDom.querySelector('article > [itemprop="publisher"]');
+  const datePublished = articleDom.querySelector('article > [itemprop="datePublished"]');
+  const identifiers = articleDom.querySelector('article > [data-itemprop="identifiers"]');
+
+  return `<div class="header">
+    ${headline?.outerHTML}
+    ${authors?.outerHTML}
+    ${affiliations?.outerHTML}
+    ${publisher?.outerHTML}
+    ${datePublished?.outerHTML}
+    ${identifiers?.outerHTML}
+  </div>`;
+}
+
+const getArticleHtmlWithoutHeader = (articleDom: DocumentFragment): string => {
+  const articleElement = articleDom.children[0];
+  console.log(articleElement.outerHTML);
+
+  let articleHtml = "";
+  articleElement.querySelectorAll('[data-itemprop="identifiers"] ~ *').forEach((elem) => articleHtml += elem.outerHTML);
+
+  return `<article itemscope="" itemtype="http://schema.org/Article" data-itemscope="root">${articleHtml}</article>`;
```

Comment:
do we need the empty `itemscope=""` here?

Reply:
I doubt it - I was attempting to recreate the article element as it is. I have no other attachment to this attribute XD